site/content/blog/2022-03-01-whats-new-in-svelte-march-2022.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "What's new in Svelte: March 2022"
+description: "Shadow endpoints and update detection come to SvelteKit"
```

Comment:
There's a discussion about whether we name this "Shadow endpoints" or not. We should adjust the title according to the outcome.